protocol-designer/src/pages/Designer/ProtocolSteps/index.tsx

```diff
@@ -84,6 +88,16 @@ export function ProtocolSteps(): JSX.Element {
         justifyContent={JUSTIFY_FLEX_START}
       >
         <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing16}>
+          {tab === 'protocolSteps' ? (
+            <Flex
+              justifyContent={JUSTIFY_CENTER}
+              width="100%"
+              paddingTop="37.5px"
+              paddingBottom="21.5px"
```

Comment:
need to use these values?

```suggestion
              paddingTop="2.34375rem"
              paddingBottom="1.34375rem"
```